Source/WTF/wtf/HashMap.h

```diff
@@ -187,6 +187,7 @@ class HashMap final {
     //   static bool equal(const ValueType&, const T&);
     //   static translate(ValueType&, const T&, unsigned hashCode);
     template<typename HashTranslator, typename K, typename V> AddResult add(K&&, V&&);
+    template<typename HashTranslator, typename K, typename Functor> AddResult ensure(K&&, Functor&&);
```

Comment:
Comments above mention only `find` and `add`, yet we have `contains`, `get`, `inlineGet`, `remove`, and `ensure`.